hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -558,6 +558,24 @@
       stage: V2
       tests: []
     }
+    {
+      name: chip_rv_dm_end_to_end_debug
+      desc: '''End to end debug test involving full GDB/OpenOCD/JTAG chain
+
+            - Load and begin executing some known binary
+            - Connect GDB via OpenOCD and JTAG to OpenTitan Earlgrey
+            - Trial the following activies within GDB, ensuring the correct
+              behaviour is seen:
+              - Interrupting and halting execution
```

Comment:
Perhaps we should say "Halting execution and resetting". I believe it's possible to halt without resetting, but I'm not sure we'd ever need or want to do it.

Reply:
@dmcardle should this also describe the bit about how to overcome the watchdog timer issue too somewhere?

Reply:
I think we can leave that out for M2 — the watchdog timer is really only an issue when we're attempting to connect to a device running a program that sets up the watchdog timer. The foolproof solution is to disable ROM execution via the OTP (or run a test program that never sets up the watchdog timer).  I'm tracking this problem in #15174.

@alphan WDYT?

Reply:
Correct.